tasks.py

```diff
@@ -709,7 +713,39 @@ def unittest(
         for individual_exclude_tag in exclude_tag:
             command += f" --tag {individual_exclude_tag}"
 
-    run_command(context, command)
+    if test_db_name:
+        run_command(context, command, test_db_name=test_db_name)
+    else:
+        run_command(context, command)
+
+
+def unittest_forked(fork_context):
+    unittest(
+        fork_context[0],
+        failfast=True,
+        label=f"nautobot.{fork_context[1]}",
+        skip_docs_build=True,
```

Comment:
This might be problematic for the tests that require docs unless you've asserted that they were built first (or only once) prior to running the forked tests.

Reply:
Looking way too far into it: This was committed as-is to get a performance comparison. The interface isn't robust at all. But I didn't want every fork rebuilding docs.

Reply:
Hey man. I want this to happen, okay! 😎